README.md

```diff
@@ -181,14 +181,14 @@ If, for some reason, the command returns a message asking for a minimum terminal
 ### Playing the game
 
 The game takes place on a rectangular arena where a snake is continuously moving (that is,
-it never stops) in one of the four directions: left, right, up and down.
-As the snake moves, it loses energy and if all of it is exausted, the
-snake dies. To recover energy, the snake needs to eat pieces of food which
+it never stops) in one of the four directions: left, right, up, and down.
```

Comment:
The added comma (oxford comma) is optional, so the old version is also correct.

But I do believe that the new version is better

Reply:
Yes, it is indeed optional. I think the added comma makes it easier to read.